### ISSUE TITLE
Can not download prebuilt images on Raspbian

### ISSUE THREAD

**Issue by Porrumentzio** [grocy/grocy-docker#103]
I get these errors when trying to download the images:
```
$ docker pull grocy/grocy
Using default tag: latest
Error response from daemon: manifest for grocy/grocy:latest not found: manifest unknown: manifest unknown
```

The error says that manifest was not found for `grocy/grocy:latest`

**Comment by Porrumentzio** [comment]
I downloaded the code from this repo and run the `docker-compose` commands, but I got this error:
![irudia](https://user-images.githubusercontent.com/61913683/98167725-d71ac080-1ee9-11eb-9ccc-70448dc05b98.png)

**Comment by Porrumentzio** [comment]
After that, I run `docker-compose build` and `docker-compose up -d` and I have the containers running.  Sorry for disturbing

**Comment by jayaddison** [comment]
Hi @Porrumentzio - no problem, thanks for mentioning this.  The initial `docker pull` problem looks like it's because there's no `latest` tag applied to `grocy-docker` images.

There's some debate about whether having a `latest` tag is a good idea or not; to begin with `grocy-docker` images are taking the approach that there are specific image versions, and within each version tag the images should correspond to a release of `grocy` itself - sometimes with minor version releases if the build process is updated.

The `exec format error` messages look like they're related to architecture differences; the images were build for `amd64` and my guess is that you're running Raspbian on an `arm` architecture of some kind?

I've opened issue #105 to track progress against building `grocy-docker` images for multiple architectures.

**Comment by Porrumentzio** [comment]
Yes, I use ARM architecture. Say me if you want me to try something out on ARM architecture!

**Comment by jayaddison** [comment]
Hi @Porrumentzio - thanks for the offer to help.  Could you try `docker pull grocy/grocy:multiarch` when you have time, and see if that works?

(this should retrieve an experimental multi-architecture build of `grocy-docker` that includes ARM platform support)

**Comment by jayaddison** [comment]
I've removed the experimental `multiarch` image since it was based on the `v3.0.1-8` Dockerfiles, and those are now out-of-date.  When a multi-architecture image is published again, we should consider adding a more descriptive version tag (and possibly automating the process?)

**Comment by jayaddison** [comment]
Hi again @Porrumentzio - this should be resolved by #156; the Docker Hub images for `grocy-docker` are now built for multiple architectures, including Arm-based systems.  Any help testing and confirming the functionality is welcome.
